diff --git a/ios/LocalizationModule.swift b/ios/LocalizationModule.swift
index 17c9cb78a927db705f331827f4f48e894317c42f..782ce5ff814fad2e882eb536e53a34633f8a46bc 100644
--- a/ios/LocalizationModule.swift
+++ b/ios/LocalizationModule.swift
@@ -123,6 +123,8 @@ public class LocalizationModule: Module {
       return "roc"
     case .iso8601:
       return "iso8601"
+    @unknown default:
+      return "unknown"
     }
   }
 
